src/entities/Discourse/utils.ts

```diff
@@ -9,4 +10,31 @@ export const DISCOURSE_USER = requiredEnv('DISCOURSE_USER')
 export const DISCOURSE_AUTH: DiscourseAuth = {
   apiKey: DISCOURSE_API_KEY,
   apiUsername: DISCOURSE_USER
-}
\ No newline at end of file
+}
+export const BASE_AVATAR_URL = requiredEnv('DISCOURSE_BASE_AVATAR_URL')
+const DEFAULT_AVATAR_SIZE = '45'
+
+function setAvatarSize(avatar_url: string) {
```

Comment:
`set` is not a proper prefix for a function that doesn't modify anything, `get` is preferred